src/components/events/GroupBuyAvalancheWidget.tsx

```
@@ -0,0 +1,215 @@
+      campaign,
+      "u-current-student"
+    );
+
+    setCampaign(updatedCampaign);
+    setCommitResult(result);
+
+    if (onCommitSuccess) onCommitSuccess(result, updatedCampaign);
+
+    if (result.isTargetReached) {
+      setNotice(
+        `🔥 GROUP BUY AVALANCHE UNLOCKED! Target hit (${updatedCampaign.currentCommitsCount}/100). All Stripe holds captured at 50% OFF ($15.00)!`
```

ESLint

[error] Insert `,`

(prettier/prettier)